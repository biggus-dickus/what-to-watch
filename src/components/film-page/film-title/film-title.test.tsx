import * as React from 'react';
import * as renderer from 'react-test-renderer';

import FilmTitle from './film-title';


it(`<FilmTitle /> should render correctly`, () => {
  const props = {
    name: `Deep Throat`,
    genre: `Films for the whole family`,
    released: 1972
  };

  expect(renderer.create(<FilmTitle {...props} />).toJSON()).toMatchSnapshot();
});
