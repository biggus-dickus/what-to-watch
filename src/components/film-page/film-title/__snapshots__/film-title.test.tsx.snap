// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FilmTitle /> should render correctly 1`] = `
Array [
  <h2
    className="movie-card__title"
  >
    Deep Throat
  </h2>,
  <p
    className="movie-card__meta"
  >
    <span
      className="movie-card__genre"
    >
      Films for the whole family
    </span>
    <span
      className="movie-card__year"
    >
      1972
    </span>
  </p>,
]
`;
